modules/images/image-loading-optimization/storage/post-type.php

```diff
@@ -0,0 +1,169 @@
+<?php
+/**
+ * Metrics storage post type.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+const ILO_PAGE_METRICS_POST_TYPE = 'ilo_page_metrics';
+
+/**
+ * Registers post type for page metrics storage.
+ *
+ * This the configuration for this post type is similar to the oembed_cache in core.
+ *
+ * @since n.e.x.t
+ * @access private
+ */
+function ilo_register_page_metrics_post_type() /*: void (in PHP 7.1) */ {
+	register_post_type(
+		ILO_PAGE_METRICS_POST_TYPE,
+		array(
+			'labels'           => array(
+				'name'          => __( 'Page Metrics', 'performance-lab' ),
+				'singular_name' => __( 'Page Metrics', 'performance-lab' ),
```

Comment:
I think this still needs to be updated per https://github.com/WordPress/performance/pull/878#discussion_r1387245125?

Reply:
Done in 9876aa